source/AsyncRgbLedHelpers.cpp

```diff
@@ -0,0 +1,70 @@
+#include "AsyncRgbLedHelpers.h"
+
+#include <cstdlib> // for memcpy
```

Comment:
`memcpy` seems to be in `<cstring>`, also add `<cassert>`, both required for gcc build

Reply:
Good catch, thanks.